src/Sentry.Hangfire/Sentry.Hangfire.csproj

```diff
@@ -16,6 +16,22 @@
     <PackageReference Include="Hangfire.Core" Version="1.8.12" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
+    <PackageReference Include="Newtonsoft.Json" Version="[5.0.1,)">
```

Comment:
Oh but we're still adding a new dependency here. Couldn't we just leave it out and suppress the warning?

Let the final app deal with it, since we don't depend on it directly.

Reply:
> Couldn't we just leave it out and suppress the warning?

If I try to add `<NoWarn>NU1903</NoWarn>` at the project level (no specific reference to this package) then we get another error message: `CS8002 Referenced assembly does not have a strong name.`. 

We also have this, but I think it's only relevant when calling `dotnet pack` (not `dotnet build`):
https://github.com/getsentry/sentry-dotnet/blob/c7b966b9a70ff504e2f06d88f34c69f1aa25b8b8/Directory.Build.props#L19-L20

The only way that I've found to suppress the warning is to add it specifically within the `PacakgeReference`... and yeah, not ideal, since we end up with an additional explicit dependency here (even though it's actually a transitive dependency).

Reply:
we need to open a ticket with Microsoft then. We shouldn't add a dependency as a workaround to disable a warning.
Rather disable warning as error on this project and keep the warning for the time being

Reply:
I'm an idiot... I was adding this: `<NoWarn>NU1903</NoWarn>` when I should have been adding `<NoWarn>$(NoWarn);NU1903</NoWarn>`. 

As a result, this was being overwritten, which was causing the `CS8002`:
https://github.com/getsentry/sentry-dotnet/blob/c7b966b9a70ff504e2f06d88f34c69f1aa25b8b8/Directory.Build.props#L16-L17

Anyway, fixed now 😝